### DISCUSSION TITLE
Difficulty with Shiny setup in Visual Studio 2019

### DISCUSSION THREAD

**Discussion by sschaub** [shinyorg/shiny#728] [Q&A] [answered]
I've tried a couple of approaches to setting up a new XF project with Shiny in VS 2019 (16.10.0), without success. I've attempted to use the Quick Start approach with the source generators, and the non-Quick Start approach. Both end up with an error in the Android project: 

Please add a reference to Mono.Android.Export.dll when using ExportAttribute or ExportFieldAttribute.	

Any tips would be appreciated!

**Comment by aritchie** [comment, marked as answer]
In your android project "Reference Manager > Assemblies > Find Mono.Android.Export.dll".

**Reply by sschaub** [reply]
Thank you for the fast response! That resolved it for me. 

For those who might encounter this later, there's a helpful screenshot in [this SO post](https://stackoverflow.com/questions/31085554/you-need-to-add-a-reference-to-mono-android-export-dll-when-you-use-exportattrib).

Wonder if it would be worth adding a note about this to the Shiny.NET [Quick Start](https://shinylib.net/quickstart) for XF newbies like me?

**Reply by aritchie** [reply]
Add it then.  I don't do feature requests on the docs.
